### PR TITLE
Swift 2.0

### PR DESCRIPTION
**Pull request by mechabyte**
Updated syntax to Swift 2.0

### CODE REVIEW

**Comment by dstayton** [comment]
Yes thanks for doing that

File: CardsAgainst.xcodeproj/project.xcworkspace/xcshareddata/CardsAgainst.xcscmblueprint [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{
```
Comment (jpsim) [inline CardsAgainst.xcodeproj/project.xcworkspace/xcshareddata/CardsAgainst.xcscmblueprint]: Could you please not check this in? See https://github.com/github/gitignore/pull/1547

**Comment by jpsim** [comment]
Please don't check in your `xcuserdata` contents.

File: CardsAgainst/View Controllers/MenuViewController.swift [hunk 2]
```diff
@@ -29,6 +29,7 @@ final class MenuViewController: UIViewController, UICollectionViewDataSource, UI
         setupStartGameButton()
         setupSeparator()
         setupCollectionView()
+        print("loading")
```
Comment (jpsim) [inline CardsAgainst/View Controllers/MenuViewController.swift]: please remove this print statement.

File: CardsAgainst/Views/TouchableLabel.swift [hunk 3]
```diff
@@ -11,32 +11,37 @@ import UIKit
...
         if let lastRange = placeholderRanges.last {
-            let mAttributedText = attributedText.mutableCopy() as NSMutableAttributedString
+            let mAttributedText = attributedText!.mutableCopy() as! NSMutableAttributedString
             mAttributedText.addAttribute(NSForegroundColorAttributeName, value: tintColor.colorWithAlphaComponent(alpha), range: lastRange)
             attributedText = mAttributedText
         }
     }
-
-    override func touchesBegan(touches: NSSet, withEvent event: UIEvent) {
+
```
Comment (jpsim) [inline CardsAgainst/Views/TouchableLabel.swift]: there's a lot of this added whitespace on empty lines. could you please revert that?

**Comment by jpsim** [comment]
This is super great, @mechabyte! Thanks for the wonderful PR :tada:. I just posted a few minor points of feedback, so once that's been addressed we'll be able to merge this.

**Comment by jpsim** [comment]
we migrated to Swift 2.1.1 in #6. Thanks for your contribution nonetheless @mechabyte!

**Pull request closed without merging**